_posts/2022-11-07-relicensing.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: Relicensing MDAnalysis
+---
+
+<a href="https://www.gnu.org/licenses/lgpl-3.0.en.html">
+<img src="https://www.gnu.org/graphics/lgplv3-with-text-154x68.png"
+title="LGPLv3" alt="LGPLv3 logo" style="float: right"/>
+</a>
+
+This blog post outlines MDAnalysis' proposal to change its license
+to the [Lesser GNU Public License][LGPL] (LGPL v3+).
+
+A summary of our [reasons for proposing this license
+change](#Rationale-for-changing-licenses), alongside upcoming actions
+for [community members and library
+contributors](#How-will-the-relicensing-process-work?) are provided.
+
+
+> ⚠️ **Disclaimer**
+> The MDAnalysis core team members are not
+> lawyers, as such the information provided here __does not, and is not
+> intended to, constitute legal advice__. This blog post also does not
+> represent MDAnalysis' full legal position on software licensing, it
+> purely aims to inform MDAnalysis developers and users on why
+> we believe the library should be relicensed.
+>
+> Further information on open-source software licensing can be found
+> from sources such as the [Open Source Initiative][OSI],
+> [tl;drLegal][tldr legal] and the [Software Sustainability Insitute][SSI licensing].
+>
+> Should you have any concerns about licensing, we always strongly
+> recommend getting legal advice before making any decisions on how
+> licensing changes may affect you.
+
+
+## Overview
+
+We want to **change the license of MDAnalysis** from the [GNU General
+Public License v2 (or any later versions)][GPLv2] (GPL v2+) to the less
+restrictive [Lesser GNU Lesser Public License v3 (or any later versions)][LGPL]
+(LGPL v3+) license. Both are [_open source licenses_][OSI] but it is
+our view that the [LGPL][] will give developers more freedom in how
+they license any of their own codes that makes use of MDAnalysis.
+
+As detailed by the [Open Source Definition][OSD], licenses are core to
+the definition of open source. "Open source doesn't just mean access
+to the source code". The license defines how code can be used, copied,
+changed, and incorporated into other code.
+
+**License changes affect how folks interact with the MDAnalysis code
+base going forward. We need the agreement of our contributors and
+community members to change from GPL to LGPL.**
```

Comment:
Being cautious in wording here, all we want is to get the approval of our devs and the community.